iot-management-device/device/src/main/kotlin/com/example/iotmanagementdevice/device/application/mapper/DeviceUpdateEventMapper.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.iotmanagementdevice.device.application.mapper
```

Comment:
why mapper in package `application.mapper`

Reply:
move it to infrastructure.kafka.mapper